src/reverb/doc/cron-for-ovh/cron-sync-orders.php

```diff
@@ -0,0 +1,16 @@
+<?php
+// Get cURL resource
+$curl = curl_init();
+// Set some options - we are passing in a useragent too here
+curl_setopt_array(
+    $curl,
+    array(
+        CURLOPT_RETURNTRANSFER => 1,
+        CURLOPT_URL => '[ADD YOUR DOMAIN URL]/modules/reverb/cron.php?code=orders',
```

Comment:
The files is added in specific in the root of the Reverb module and yes, the seller must be to configure his domain url for this specific configuration for OVH to bypass the OVH cron system. a file calls with cURL a domain url with parameters in GET (code=XXXX).
The seller doesn't use an URL who is not his URL.
It's specific for OVH (https://www.ovh.com/fr/), they have a cron system limited with no parameters in GET and the cron.php used in the Reverb module need to have parameters in GET !

If you want, the cron guide in OVH:
http://docs.ovh.com/fr/fr/web/hosting/mutualise-taches-automatisees-cron/